scripts/clean-all.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-npm install lerna \
-lerna clean --yes \
-rm -rf node_modules
```

Comment:
Guess it was mainly for the shrinkwrap thing seems like we don't need it anymore with yarn install --force indeed